src/SampleLinter/MigrationSettingsView.xaml

```diff
@@ -0,0 +1,12 @@
+﻿<Window x:Class="SampleLinter.MigrationSettingsView"
```

Comment:
I'm guessing MigrationSettings is not part of this PR ? Or am I missing something

Reply:
my mistake